fence/blueprints/data/indexd.py

```diff
@@ -314,17 +314,18 @@ def init_multipart_upload(key, expires_in=None):
         Returns:
             uploadId(str)
         """
-        try:
-            bucket = flask.current_app.config["DATA_UPLOAD_BUCKET"]
-        except KeyError:
-            raise InternalError(
-                "fence not configured with data upload bucket; can't create signed URL"
-            )
+        if bucket is None:
```

Comment:
```suggestion
        if bucket:
```

If you use Truthiness, you don't have to worry about `None` being forced instead of empty strings